### PR TITLE
Add data dictionary generation

### PR DESCRIPTION
**Pull request by gavignon**
- Add sfdc-generate-data-dictionary dependency
- Add new task generate-data-dictionary

### CODE REVIEW

**Comment by scolladon** [comment]
I think you should improve the readme with the new tool (description and references) before This pull request can be accepted

**Comment by gavignon** [comment]
Yes, it should be good now. I have also added configuration properties.

File: README.md [hunk 1]
```diff
@@ -47,15 +47,17 @@ SFDC-ci-toolkit comes with handy npm scripts for CI :
...
 }
 ```
 Combined them smartly according to your need as a developer or as a release manager ;)
 
-Here is the list of scripts with their description available in the toolkit 
+Here is the list of scripts with their description available in the toolkit
 * **coverage** : Run it just after having deployed with the RunLocalTests. It will gather the coverage and translate it to lcov format into coverage.json file
 * **deploy** : Run it to deploy your repo to Salesforce
 * **generate-package** : Run it to generate your package.xml from your repository
+* **generate-daata-dictionary** : Run it to generate your data dictionary from your Salesforce org
```
Comment (scolladon) [inline README.md]: I think there is a typo here @gavignon

File: README.md [hunk 2]
```diff
@@ -268,6 +270,8 @@ Here is the list of optional parameters with their default value :
 * SF_ROLLBACKONERROR : (boolean) used in deploy. Default: true
 * SF_SINGLEPACKAGE : (boolean) used in deploy. Default: true
 * SF_VERBOSE : (boolean) used in deploy. Default: true
+* SF_PROJECT : project name (string) used in data dictionary filename prefix
+* SF_CUSTOMOBJECTS : (boolean) used in data dictionary. Default: true
```
Comment (scolladon) [inline README.md:274]: I think this configuration var should be in the optional section

File: README.md (hunk 1 above)
Comment (gavignon) [reply README.md]: My bad, fixed.

File: README.md (hunk 2 above)
Comment (gavignon) [reply README.md:274]: Isn't it already the case ?

**Comment by gavignon** [comment]
Pull request rebased and updated

File: README.md (hunk 2 above)
Comment (scolladon) [reply README.md:274]: Yep sorry

**Review by scolladon — APPROVED** [review]

**Pull request merged**